tests/Resources/ti.ui.scrollableview.addontest.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-2018 by Appcelerator, Inc. All Rights Reserved.
```

Comment:
2020

Reply:
Moved the test cases inside ti.ui.scrollableview.js and updated.